backend/Dockerfile

```diff
@@ -61,16 +61,21 @@ ARG ENABLE_NDS_SUPPORT
 ARG ENABLE_PS1_SUPPORT
 ARG ENABLE_WII_GC_SUPPORT
 RUN if [ "${ENABLE_NDS_SUPPORT}" = "YES" ] || \
-       [ "${ENABLE_PS1_SUPPORT}" = "YES" ] || \
-       [ "${ENABLE_WII_GC_SUPPORT}" = "YES" ]; then \
+    [ "${ENABLE_PS1_SUPPORT}" = "YES" ] || \
+    [ "${ENABLE_WII_GC_SUPPORT}" = "YES" ]; then \
     dpkg --add-architecture i386 && apt-get update && \
     apt-get install -y -o APT::Immediate-Configure=false \
-        wine; \
+    wine; \
     fi
 
 # ps1 specifics
 RUN if [ "${ENABLE_PS1_SUPPORT}" = "YES" ] ; then \
     apt-get install -y dos2unix; \
+    # all the following is required to run the MS-DOS toolchain
```

Comment:
can this be `&&`'d together to ensure only 1 layer is created rather than 4? something like this (untested):

```
RUN if [ "${ENABLE_PS1_SUPPORT}" = "YES" ] ; then \
    apt-get install -y software-properties-common \
    && add-apt-repository -y ppa:dosemu2/ppa \
    && apt-get update \
    && apt-get install -y dos2unix dosemu2 \
fi
```